docs/release-notes/1172-fix-dashboard-limit.txt

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+**Bug Fixes**
+
+- Reapply the filtering for just the dashboard recent tasks. The limit filter was
```

Comment:
blocking: I think for bugfixes we don't need so much detail. "Fix an issue where the Dashboard task list did not apply the limit filter properly."